### ISSUE TITLE
how to remove faces textured with empty-color

### ISSUE THREAD

**Issue by nkennek** [cdcseacave/openMVS#576]
As introduced [wiki](https://github.com/cdcseacave/openMVS/wiki/Usage), faces that are not visible in any of the input images appear in orange with default parameters.
I figured out they can be colored differently, but do not see how to remove them (although the introduction says they can be colored or removed).
Is there any way to do this by options?

thanks in advance,

**Comment by cdcseacave** [comment]
no, there is no param for this, though should be relatively easy to add; feel free to contribute, in texture module at the end are managed the invisible triangles, instead of asigning the fix color you can simply remove them.

**Comment by CanCanZeng** [comment]
Hi @nkennek , did you solve this problem? I also think it is better to remove the empty-color pixels, is it right to do my own logic here? https://github.com/cdcseacave/openMVS/blob/a69a52c09587a1dae82833fe06feac3ac40da5ba/libs/MVS/SceneTexture.cpp#L1943

**Comment by cdcseacave** [comment]
Closing as stale. The codebase has been substantially updated on the \develop\ branch. Please reopen with reproduction steps if the issue persists on the latest code.
